amplify.yml

```diff
@@ -6,8 +6,8 @@ frontend:
         - nvm install 18.13
         - nvm use 18.13
         - |
-          if [[ "${AWS_BRANCH_ARN#*/branches/}" == *"pr-"* ]]; then
-            PREVIEW_URL="https://${AWS_BRANCH_ARN#*/branches/}.${AWS_APP_ID}.amplifyapp.com";
+          if [[ "${AWS_BRANCH}" != "main" && "${AWS_BRANCH}" != "staging" ]]; then
+            PREVIEW_URL="https://${AWS_BRANCH}.${AWS_APP_ID}.amplifyapp.com";
```

Comment:
Instead of removing the existing PR logic, let's leave it in and add a else if `elif` command as other organizations will have the normal previews and they might not have the branches `main` and `staging`.